### ISSUE TITLE
Nothing is rendered until window is resized (Mac)

### ISSUE THREAD

**Issue by galvinben** [liamg/aminal#29]
**Describe the bug**
When first running aminal, the window loads, but nothing is rendered until the window is resized

**To Reproduce**
1. run `./aminal`
2. See that nothing is rendered.
3. Resize window and see that it renders.

**Expected behavior**
Should render without having to resize the windown.

**Environment (please complete the following information):**
 - macOS Mojave 10.14
 - go version go1.11 darwin/amd64

**Logs**

```
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:50   Initiating GUI resize to 506x302
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:55   Updating font resolution...
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:60   Setting renderer area...
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:63   Calculating size in cols/rows...
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:66   Resizing internal terminal...
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:71   Setting viewport size...
2018-10-23T16:08:09.087+0100    DEBUG   gui/gui.go:74   Resize complete!
```

**Comment by liamg** [comment]
This is because the framebuffer size and window size are different on OSX. This is also the cause of everything rendering really small on OSX at the moment. Just needs the window size call replacing with a frame buffer size call before the main render loop.

**Comment by liamg** [comment]
Also need to modify the resize syscall for Darwin to use 
```
TIOCGWINSZ_OSX = 1074295912
```

**Comment by jiegec** [comment]
Still observed in commit b909cb625dbc05596c0c214c4032909b91683c1e.
Running macOS Mojave 10.14.1 with go go version go1.11.2 darwin/amd64.

**Comment by vkravets** [comment]
Fixed at #102 and #104

**Comment by jiegec** [comment]
I can confirm that it is fixed in master now. Thanks for that.
